### DISCUSSION TITLE
ArgoCD does not apply the job in sub-chart during installtion

### DISCUSSION THREAD

**Discussion by becash143** [argoproj/argo-cd#9149] [General]
I was deploying my charts using ArgoCD, however the job inside my airflow template is not running. But with helm install, I could  see that job running. What could be the reason behind it or is there any configurations I am missing in ArgoCD?

**Comment by todaywasawesome** [comment]
@becash143 I assume you don't have any filters on the application definition? I would double check that it's syncing correctly, run `helm template` and review the output etc.

**Reply by becash143** [reply]
@todaywasawesome The issue is with hooks used in jobs. May be ArgoCD interprets the hooks differently. After removing the hooks, the deployment was successful.
```
annotations:
    "helm.sh/hook": post-install,post-upgrade
    "helm.sh/hook-weight": "3"
    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
```
